test_controllingChargebyteBoard.py

```diff
@@ -0,0 +1,68 @@
+from controllingChargebyteBoard import *
+
+
+class Test:
```

Comment:
hmmm, the problem is, that the pytest classes all should include test in the name, so this would be confused with the unit test class.... what about ChargebyteBoard ?